test/unit/test.speech_to_text.v1.js

```diff
@@ -223,6 +223,27 @@ describe('speech_to_text', function() {
     });
   });
 
+  describe('recognizeWebM()', function() {
+    const session_path = '/v1/sessions/foo/recognize';
+    const payload = {
+      audio: fs.createReadStream(__dirname + '/../resources/sample1.webm'),
+      content_type: 'audio/webm'
+    };
+
+    it('should generate a valid payload with session', function() {
+      const req = speech_to_text.recognize(extend({ session_id: 'foo' }, payload), noop);
+      assert.equal(req.uri.href, service.url + session_path);
+      assert.equal(req.method, 'POST');
+      assert.equal(req.headers['Content-Type'], payload.content_type);
```

Comment:
This is good, but you should add another test that doesn't set the content_type, and then asserts that it is correctly detected as `audio/webm`.